cpu/cc430/cc430-adc.c

```diff
@@ -71,7 +71,6 @@ uint16_t adc12_single_conversion(uint16_t ref, uint16_t sht, uint16_t channel)
     ADC12CTL1 = ADC12SHP;                       /* Enable sample timer */
     ADC12MCTL0 = ADC12SREF_1 + channel;         /* ADC input channel   */
     ADC12IE = 0x001;                            /* ADC_IFG upon conv result-ADCMEMO */
-    eINT();
```

Comment:
Can someone with knowledge of the chronos board validate this? (This function is called by the function `battery_get_voltage`, except `battery_get_voltage` is never used anywhere in-tree)